docs/config.md

```diff
@@ -61,20 +59,22 @@ interfaces
 The `participant` needs to be the same as the one specified in the `preciceConfig`,
 which is the main preCICE configuration file. The `preciceConfig` can be a path and needs to be wrapped with quotation marks.
 
-The list `modules` can contain `CHT` or/and `FSI` (separated by space).
+The list `modules` can contain `CHT`, `FSI`, or/and `FF` (separated by space).
+Each module provides some fields common in the respective type of simulation, and fields from different modules can be combined.
+For example, this is a valid combination: `modules (CHT FF);`.
 
 In the `interfaces`, we specify the coupling interfaces (here only one).
 The `mesh` needs to be the same as the one specified in the `preciceConfig`.
 The `patches` specifies a list of the names of the OpenFOAM boundary patches that are
 participating in the coupled simulation. These need to be defined in the files
-included in the `0/` directory. The names of the interfaces (e.g. `Interface1`) are arbitrary and are not used.
+included in the `0/` directory. The names of the interfaces (e.g., `Interface1`) are arbitrary and are not used.
 
-The `locations` field is optional and its default value is `faceCenters` (with `faceCentres` also accepted), signifying that the interface mesh is defined on the cell face centers. The alternative option is `faceNodes`, which defines the mesh on the face nodes and is needed e.g. for reading displacements in an FSI scenario.
+The `locations` field is optional and its default value is `faceCenters` (with `faceCentres` also accepted), signifying that the interface mesh is defined on the cell face centers. The alternative option is `faceNodes`, which defines the mesh on the face nodes and is needed, e.g., for reading displacements in an FSI scenario.
```

Comment:
```suggestion
The `locations` field is optional and its default value is `faceCenters` (with `faceCentres` also accepted), signifying that the interface mesh is defined on the cell face centers. The alternative option is `faceNodes`, which defines the mesh on the face nodes, is needed, e.g., for reading displacements in an FSI scenario.
```

Reply:
I think this is not what you meant to fix, as it now makes less sense to me.

Reply:
Ah no, you are right.